epochStart/metachain/trigger_test.go

```diff
@@ -175,7 +175,7 @@ func TestTrigger_ForceEpochStartCloseToNormalEpochStartShouldNotForce(t *testing
 	epochStartTrigger.currentRound = 20
 
 	epochStartTrigger.ForceEpochStart(201)
-	assert.Equal(t, uint64(math.MaxInt64), epochStartTrigger.nextEpochStartRound)
+	assert.Equal(t, uint64(math.MaxUint64), epochStartTrigger.nextEpochStartRound)
```

Comment:
actually this will cause an error from the compiler stating the following:
```GO
./trigger_test.go:178:14: constant 18446744073709551615 overflows int
```